base_layer/wallet/src/key_manager_service/storage/sqlite_db/key_manager_state.rs

```diff
@@ -85,17 +88,22 @@ impl NewKeyManagerStateSql {
 }
 
 impl KeyManagerStateSql {
+    /// Retrieve every key manager branch currently in the database.
+    /// Will always return a vec of [KeyManagerStateSql], if none are found, it will return an empty vec
```

Comment:
I don't think this is quite correct, it will return an error if the load fails.
```suggestion
    /// Returns a `Vec` of [KeyManagerStateSql], if none are found, it will return an empty `Vec`.
```